test/config/multiplesettest.go

```diff
@@ -50,20 +46,14 @@ func (s *TestSuite) TestMultipleSet(t *testing.T) {
 
 		// Set a value using gNMI client
 		targetPath := gnmi.GetTargetPathWithValue(simulator.Name(), tzPath, msValue, proto.StringVal)
-		transactionID, transactionIndex := gnmi.SetGNMIValueOrFail(t, gnmiClient, targetPath, gnmi.NoPaths, gnmi.NoExtensions)
+		transactionID, transactionIndex := gnmi.SetGNMIValueOrFail(t, gnmiClient, targetPath, gnmi.NoPaths, gnmi.SyncExtension(t))
 		assert.NotNil(t, transactionID, transactionIndex)
 
-		err := gnmi.WaitForConfigurationCompleteOrFail(t, configapi.ConfigurationID(simulator.Name()), time.Minute)
-		assert.NoError(t, err)
-
 		// Check that the value was set correctly
 		gnmi.CheckGNMIValue(t, gnmiClient, targetPath, msValue, 0, "Query after set returned the wrong value")
 
 		// Remove the path we added
-		gnmi.SetGNMIValueOrFail(t, gnmiClient, gnmi.NoPaths, targetPath, gnmi.NoExtensions)
-
-		err = gnmi.WaitForConfigurationCompleteOrFail(t, configapi.ConfigurationID(simulator.Name()), time.Minute)
-		assert.NoError(t, err)
+		gnmi.SetGNMIValueOrFail(t, gnmiClient, gnmi.NoPaths, targetPath, gnmi.SyncExtension(t))
 
 		//  Make sure it got removed
 		gnmi.CheckGNMIValue(t, gnmiClient, targetPath, "", 0, "incorrect value found for path /system/clock/config/timezone-name after delete")
```

Comment:
It is not clear to me this function is checking onos-config or target. If it is onos-config is good but perhaps we should check the target as well to make sure the value is updated properly.

Reply:
This test does not interrogate the target

Reply:
That would be a good thing to add.

Reply:
My point was we should no? It is a multi set thing and we should make sure it is applied to the target eventually. if onos-config has the change doesn't mean it is on the target.